CHANGES.md

```diff
@@ -1,6 +1,9 @@
 ## Changes in 0.5.0.devX (in development)
 
-* From 0.4.1: Fixed time-series performance drop (#299).
+### Fixes
+* From 0.4.1: Fixed time-series performance drop (#299). 
+* Fixed incorrect handling input slices with defined packing of variables when inserting 
```

Comment:
Missing a context here: Users have no idea where "incorrect handling input slice" will apply to. Say something like:

* Fixed `xcube gen` CLI tool to correcly insert time slices into an existing cube stored as Zarr.